include/picongpu/plugins/output/WriteSpeciesCommon.hpp

```diff
@@ -58,6 +58,8 @@ namespace picongpu
             {
 #if(PMACC_CUDA_ENABLED == 1)
                 CUDA_CHECK((cuplaError_t) cudaHostAlloc(&ptr, size * sizeof(type), cudaHostAllocMapped));
+#elif(ALPAKA_ACC_GPU_HIP_ENABLED == 1)
+                CUDA_CHECK((cuplaError_t) hipHostMalloc((void**) &ptr, size * sizeof(type), hipHostRegisterMapped));
```

Comment:
This flag seems to not match its CUDA counterpart. Perhaps worth checking and explaining, as the same question would arise later when unifying this piece for CUDA and HIP.

Reply:
There is no `hipHostAllocMapped`, all I found is that `hipHostRegisterMapped` is a flag to use. The documentation for `hipHostMalloc` is very bad :-(.